hr_employee_ppe/migrations/13.0.2.0.0/post-migration.py

```diff
@@ -47,7 +47,7 @@ def migrate(env, version):
             state,
             product_id,
             start_date,
-            expiry_date,
+            end_date,
```

Comment:
The problem is only located in the ppe table, so the only change should only appear in line 64